storage/index/options.go

```diff
@@ -33,11 +33,15 @@ import (
 const (
 	// defaultIndexInsertMode sets the default indexing mode to synchronous.
 	defaultIndexInsertMode = InsertSync
+
+	defaultTagArrayPoolInitCapacity = 5
+	defaultTagArrayPoolMaxCapacity  = 20
```

Comment:
nit: Perhaps 8 and 32 just to stay in base 2?

Reply:
sure

Reply:
done